tests/subsys/lorawan/clock_sync/testcase.yaml

```diff
@@ -10,5 +10,6 @@ tests:
   lorawan.clock_sync.phy:
     depends_on: lora
     filter: CONFIG_ENTROPY_HAS_DRIVER
+    timeout: 500
```

Comment:
I think this probably deserves the `slow: true` flag.

Reply:
Didn't know that exists, but yeah, that would make a lot of sense. It would even qualify for `very_slow` ;)